src/api/object_api.js

```diff
@@ -1373,6 +1374,9 @@ module.exports = {
                 is_latest: { type: 'boolean' },
                 delete_marker: { type: 'boolean' },
                 num_parts: { type: 'integer' },
+                num_multiparts: { type: 'integer' },
+                multipart_start: { type: 'integer' },
+                multipart_end: { type: 'integer' },
```

Comment:
You can add minimum and maximum validation:
```
"minimum": 1,
"maximum": 1000
```
https://json-schema.org/understanding-json-schema/reference/numeric.html